doc/source/docs/reference/geodataframe.rst

```diff
@@ -59,12 +59,22 @@ Aggregating and exploding
    GeoDataFrame.dissolve
    GeoDataFrame.explode
 
+Overlay operations
+------------------
+
+.. autosummary::
+   :toctree: api/
+
+   GeoDataFrame.clip
+   GeoDataFrame.overlay
+   GeoDataFrame.sjoin_nearest
```

Comment:
`sjoin_nearest` doesn't seem to fit well here (ditto your comment about `sjoin` in #1984)

Reply:
What about splitting it into Overlay operations with `overlay` and `clip` and Spatial joins with `sjoin` and `sjoin_nearest`?

Reply:
Done. I have included `sjoin` in there and removed the same from #1984 to avoid conflicts.